scripts/broker.csx

```diff
@@ -0,0 +1,104 @@
+#r "System.Data"
+#r "../src/packages/RabbitMQ.Client.4.1.1/lib/net451/RabbitMQ.Client.dll"
```

Comment:
At least when simple-targets is updated, you're probably thinking in the context of the build script already, vs a rabbit client update.

Not really anything to be done about it, but something to make sure we remember.